_forms/threshold.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: form
+title: "Threshold"
+title_mark: Threshold
+published_date: 2025-11-01
+refactored_date: 2025-11-01
+authors:
+  - michael-basil
+  - kyle-ingersoll
+excerpt: "A threshold is the line between thought and action, Visionary and Driver energy."
+---
+
+## Intention
+
+A threshold is the line between thought and action, Visionary and Driver energy. It is when the body is forced to act and make a decision. As Koichi Tohei says, however, Mind moves Body. Thus the decision will be based on one's values, attitudes, and general conditioning.
+
+## Experience
+
+You will feel the tension as you get close to the threshold. However, to refuse to cross a threshold one has set out for you, is to cross a threshold you set out for yourself. Thus, one can never truly refuse to cross a threshold, doing nothing is still a decision, thus one can only choose which thresholds to cross. We are always crossing thresholds in their life, whether we like it or not.
+
+## Implementation
+
+You first ensure that the other person understands the full context of the choice they are about to make, to the best of your ability. While we cannot fully convey our context to other people through language and we all operate on incomplete information, as a rule of thumb, don't manipulate people. Then after laying out the threshold, the door, you invite them to cross through while making it clear the consequences if they don't and that you are willing to follow through on the consequences. As Chris Voss says, Never Split the Difference (never make a bad deal) since it is worse than no deal. If they cross through, great, if they don't, respect that decision.
```

Comment:
I would break this into two parts ...

When offering ... (Also note that when you offer a threshold for someone, you are now part of that field and crossing through should they decide to engage it, so there is a responsibility here)

When encountering ... (regardless of someone else is offering or it has somehow become apparent) ..